packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -4,6 +4,8 @@ import last from 'lodash-es/last';
 import { useTranslation } from 'react-i18next';
 import { useLocation } from 'react-router-dom';
 import { ConfigurableLink } from '@openmrs/esm-framework';
+import { ActivityIcon } from '@openmrs/esm-framework';
+import styling from './dashboardextension.scss';
```

Comment:
```suggestion
import styles from './dashboardextension.scss';
```

Can we rename this file to `dashboard-extension.scss` for consistency? https://o3-docs.openmrs.org/docs/coding-conventions#naming